client/state/selectors/test/get-follow-count.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getFollowCount
+} from '../';
+
+describe( 'getFollowCount()', () => {
```

Comment:
I wonder if we should hook up selector tests to the reducers so that they won't go out of sync.

All the tests I've seen within `client/state/selectors` seem to be essentially a test on `_.get(..)` which might be less useful.